### ISSUE TITLE
Store items not saving positions on Game Saved

### ISSUE THREAD

**Issue by RedJohn260** [Krutonium/KruFPS#16]
Store items like juice bottles, sausages, milk, sugar, etc... are not saving world positions when the game is saved. When loading the game, items returns to original positions.

**Comment by RedJohn260** [comment]
However, upon further playing the game, i discovered that this seems to be related to issue #17

**Comment by Krutonium** [comment]
Have you tested this with current source?

**Comment by RedJohn260** [comment]
No, not with the latest merge. I'll give it another try tomorrow.

**Comment by Athlon007** [comment]
I found that issue yesterday and fixed it with latest update. It should be OK now
